compiler/rustc_metadata/src/locator.rs

```diff
@@ -244,7 +245,7 @@ crate struct CrateLocator<'a> {
 
     // Immutable per-search configuration.
     crate_name: Symbol,
-    exact_paths: Vec<PathBuf>,
+    paths: Vec<CanonicalizedPath>,
```

Comment:
Could you keep the old name?
The prefix is important, this field determines the choice between two different search modes - "use exact paths" and "walk search directories".